src/modules/QtQuick/Animation.js

```diff
@@ -1,19 +1,21 @@
-QmlWeb.registerQmlType({
-  module: "QtQuick",
-  name: "Animation",
-  versions: /.*/,
-  baseClass: "QtQml.QtObject",
-  enums: {
+QmlWeb.registerQmlType(class {
+  static module = "QtQuick";
+  static element = "Animation";
```

Comment:
@akreuzkamp We could do both — use `.name` by default and write `class Animation {` in our code, but allow users to override it with a static property so that they could use anonymous functions there. How should that property be named then?